kglm/tests/models/alias_copynet_test.py

```diff
@@ -1,21 +1,19 @@
 # pylint: disable=protected-access,not-callable,unused-import
 
-from allennlp.common.testing import ModelTestCase
+from kglm.common.testing import KglmModelTestCase
 import numpy as np
 import torch
 
 from kglm.data.dataset_readers.enhanced_wikitext import EnhancedWikitextKglmReader
 from kglm.models.alias_copynet import AliasCopynet
 
 
-class  AliasCopynetTest(ModelTestCase):
+class AliasCopynetTest(KglmModelTestCase):
 
     def setUp(self):
         super().setUp()
         self.set_up_model("kglm/tests/fixtures/training_config/alias_copynet.json",
                           "kglm/tests/fixtures/enhanced-wikitext.jsonl")
 
     def test_model_can_train_save_and_load(self):
-        # TODO: Construct a test case where we can ensure that _copy_mode_projection is learning
```

Comment:
@rloganiv i hope this comment doesn't apply anymore...it seems to calculate grads just fine for `_copy_mode_projection`. Hopefully this is expected?

Reply:
(for posterity, seems like the answer is yes).